src/seqdisp.cpp

```diff
@@ -871,35 +869,33 @@ static bool seq_AddTextFromFile(const char *pTextName, SEQ_TEXT_POSITIONING text
 	}
 
 	pTextBuffer = fileLoadBuffer;
-	pCurrentLine = strtok(pTextBuffer, seps);
-	while (pCurrentLine != nullptr)
+	std::istringstream stream(pTextBuffer);
```

Comment:
We should probably add something like:
```cpp
stream.imbue(std::locale::classic());
```
before the loop, to ensure that the global locale does not impact parsing